src/components/User/TeamInfo.jsx

```diff
@@ -0,0 +1,37 @@
+import Input from "../Input";
+
+const TeamInfo = ({ user, team }) => {
+  return user ? (
+    <div>
+      <Input
+        name="github"
+        type="text"
+        title="Github Link"
+        value={user.github}
+        user={user}
+        editable={true}
+      />
+      <Input
+        name="devpost"
+        type="text"
+        title="Devpost Link"
+        value={user.devpost}
+        user={user}
+        editable={true}
+      />
+
+      <div>
+        {team.map((member, memberIndex) => (
+          <div key={memberIndex}>
+            <p>TEAM MEMBER {memberIndex + 1}</p>
+            <p>{member.members[memberIndex]}</p>
```

Comment:
why is this needed? isn't member already an array of members?

Reply:
it should be team.members.map

Reply:
team.members.map creates an error "Cannot read properties of undefined (reading 'map')"